<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629682815793' stop='1629682955907'>
    <name>verify image project append funtion</name>
    <title>verify image project append funtion</title>
    <test-cases>
        <test-case start='1629682815793' status='passed' stop='1629682818427'>
            <name>Qick append verify add new line</name>
            <title>Qick append verify add new line</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1629682818427' status='failed' stop='1629682946523'>
            <name>Qick append verify delete new line</name>
            <title>Qick append verify delete new line</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='3aabc454-5e28-4b94-8806-db8b48f8e5c6-attachment.png' type='image/png' size='45532'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629682946524' status='failed' stop='1629682947825'>
            <name>Qick append verify publish</name>
            <title>Qick append verify publish</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a2c55c68-5425-4eca-b811-533ca530bc41-attachment.png' type='image/png' size='45532'/>
            </attachments>
            <failure>
                <message>Failed: No element found using locator: By(css selector, #uploadImage0)</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(css selector, #uploadImage0)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:814:27
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)Error: 
                    at ElementArrayFinder.applyAction_ (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as sendKeys] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as sendKeys] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:831:22)
                    at CommonAppend.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\general\common-append.ts:69:32)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                From: Task: Run it("Qick append verify publish") in control flow
                    at UserContext.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:94:19)
                    at attempt (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error: 
                    at Suite.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-image-spec.ts:26:3)
                    at addSpecsToSuite (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\append-project-image-spec.ts:10:1)
                    at Module._compile (node:internal/modules/cjs/loader:1101:14)
                    at Module.m._compile (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:422:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:425:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629682947825' status='passed' stop='1629682955905'>
            <name>File append verify selete existing file</name>
            <title>File append verify selete existing file</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
    </test-cases>
</ns2:test-suite>